collectors/python.d.plugin/alarms/metadata.yaml

```diff
@@ -68,10 +140,10 @@ metrics:
       unit: "status"
```

Comment:
I can't comment in the particular line, GitHub wont let me, can you add `These metrics refer to the entire monitored application.` on the global scope descriptions? It is a standard wording we use (mainly to explain clearly on multiple scopes, but even when only one exists it is a nice explanation to have)

please check for all 3 files, probably fail2ban needs this addition too